lisa/src/main/java/it/unive/lisa/program/cfg/statement/CFGCall.java

```diff
@@ -203,25 +204,34 @@ public final Identifier getMetaVariable() {
 		// metavariable
 		AnalysisState<A, H, V> returned = callGraph.getAbstractResultOf(this, lastPostState, params);
 		// the lub will include the metavariable inside the state
-		AnalysisState<A, H, V> lub = lastPostState.lub(returned);
-
-		if (getStaticType().isVoidType())
+		// AnalysisState<A, H, V> lub = lastPostState.lub(returned);
+
+		if (getStaticType().isVoidType() ||
+		// FIXME: @Luca, the problem here is that we have an untyped that indeed
```

Comment:
The heuristic you went for here looks good: if there are other situations that evade such check, we could enforce the type inference...